beacon-chain/db/kv/blocks.go

```diff
@@ -4,29 +4,29 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/boltdb/bolt"
 	"github.com/gogo/protobuf/proto"
+	"github.com/karlseguin/ccache"
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/go-ssz"
 	"github.com/prysmaticlabs/prysm/beacon-chain/db/filters"
 	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
-	"github.com/prysmaticlabs/prysm/shared/bytesutil"
 	"github.com/prysmaticlabs/prysm/shared/sliceutil"
 	"go.opencensus.io/trace"
 )
 
+var blockCache = ccache.New(ccache.Configure())
```

Comment:
Is this OK? The default is 5000 objects.

https://github.com/karlseguin/ccache/blob/ec06cd93a07565b373789b0078ba88fe697fddd9/configuration.go#L24